dracut/image-boot/eos-map-image-file

```diff
@@ -26,7 +26,11 @@ if [ "$#" != "3" ]; then
   exit 1
 fi
 
-if ! fstype=$(lsblk --noheadings -o FSTYPE "${host_device}"); then
+# lsblk can output many lines, we only want the first. However, we use
```

Comment:
Why does it output many lines? would `lsblk --nodeps` do the right thing, without needing to murge its output like this?

Reply:
Ah, that looks like it'll do nicely.